.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+dist
```

Comment:
Move this to a new PR, this one needs to be discussed more. Having `dist` on the repo sucks but we did it anyway so that storybook could be installed from the Github repo